src/foam/u2/Element.js

```diff
@@ -2128,10 +2128,19 @@ foam.CLASS({
       name: 'attribute',
       value: false
     },
+    {
+      class: 'String',
+      name: 'placeholder'
```

Comment:
I've seen us try to set `placeholder` on a property in a few spots or override the view to include a placeholder so this seems like it would be pretty convenient to have.